app/views/content_items/service_sign_in/_choose_sign_in.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render "components/back-link", href: @content_item.back_link %>
+<%= render "components/back_link", href: @content_item.back_link %>
```

Comment:
Gems in the component _ought_ to be using hyphens too. It's on my todo list to update the existing ones. They incorrectly copied the older Static component conventions.

Reply:
Though `govuk_publishing_components/components/back-link` would be an awkward mix of hyphens and underscores. 🤔

Reply:
We're already using hyphens when importing the `scss` files, so I think you're right, we ought to use hyphens everywhere. (This has already tripped me up a couple of times)

I don't think we include the name of the gem in any `import` or `render` lines, so hopefully there won't be many places where there's a mish-mash of underscores and hyphens.

Do you want me to update the gem now to use hyphens for all components? I suppose that'd need to be a major version bump.